src/main/java/uk/gov/dwp/dataworks/provider/hsm/HsmDataKeyDecryptionConstants.java

```diff
@@ -9,7 +9,8 @@
     String PADDING = "OAEPPadding";
     String SYMMETRIC_KEY_TYPE = "AES";
     String DATA_KEY_LABEL = "data_key";
-    String PROVIDER = "Cavium";
+    String CAVIUM_PROVIDER = "Cavium";
+    String BC_PROVIDER = "BC";
```

Comment:
Is this still needed or was it from the first try?

Reply:
Removed, thanks.